components/jwt-sidecar/run-sidecar.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 
 /usr/local/bin/execute-get-token.sh &
-/usr/local/bin/execute-get-vault-secrets.sh
+/usr/local/bin/execute-get-vault-secrets.sh &
+usr/local/bin/execute-get-keycloak-identities.sh
```

Comment:
typo prefix `/`? hmm  did this work? I would prefer blocking on `get-vault-secrets.sh` instead since its more stable for us.

Reply:
Ha! It worked becauase the working directory is "/" :D

Reply:
I think we have to block on both since only one of them can be provided, both or none.